modules/libdeflate/1.18/patches/add_build_file.patch

```diff
@@ -0,0 +1,52 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,49 @@
++# SPDX-License-Identifier: BSD-3-Clause
++# Copyright (c) Contributors to the OpenEXR Project.
++
++cc_library(
++    name = "deflate",
```

Comment:
If you add an alias pointing to this target with name `libdeflate`, then users can depend on this library as just `@libdeflate` instead of `@libdeflate//:deflate`. Not a requirement, we just recommend this so that users can usually guess how to add a dep on a single-target library.

Reply:
Added an alias